pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -103,7 +103,7 @@ func deletePodHandler(c clientset.Interface, emitEventFunc func(types.Namespaced
 	return func(ctx context.Context, args *WorkArgs) error {
 		ns := args.NamespacedName.Namespace
 		name := args.NamespacedName.Name
-		klog.V(0).InfoS("NoExecuteTaintManager is deleting pod", "pod", args.NamespacedName.String())
```

Comment:
Is this a new recommended thing to do with logs? Why make this change?

Reply:
@bowei  klog.V(0).InfoS is equal to klog.InfoS, V(0) unnecessary